web_server_local.py

```diff
@@ -28,4 +28,4 @@
 
 handler_class = partial(SimpleHTTPRequestHandler, directory=directory)  # Python 3.7+
 print(f'Update channel: {settings.build_name}')
-test(HandlerClass=handler_class, port=args.port, bind='localhost')
+test(HandlerClass=handler_class, port=args.port, bind='127.0.0.1')
```

Comment:
What is wrong here?
I know that ip6 is default, but why it is not good for you?

Reply:
I don't mind IPv6, I mind different addresses depending on version of Python. IITC-Button now uses 127.0.0.1 and requires a manual change to [::1] in settings. It's not good.

Reply:
You can use `localhost` to cover both

Reply:
Ok, you are right